cmake/README.md

```diff
@@ -169,29 +200,6 @@ cmake -DTBB_TEST_SPEC=ON ..
 make test_spec
 ```
 
-## Intallation and packaging
```

Comment:
What do you think about ref to INSTALL.md here?

Reply:
done